maquetado/styles.css

```diff
@@ -61,6 +61,37 @@ body {
   color: #999;
 }
 
+.footer-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+  align-content: center;
+  padding: 10px 0;
+  background-color: #f5c85f;
+  color: white;
+}
+
+.subscribe {
+  height: 40px;
+  width: 100px;
+  margin: 25px 0;
+  color: white;
+  background-color: transparent;
+  border: 1px solid white;
+  border-radius: 10px;
+  font-size: 1.1em;
```

Comment:
we tend to use pixels for font sizes, this is to have more control over the size of things.